owncloudApp/src/main/java/com/owncloud/android/authentication/PassCodeManager.java

```diff
@@ -57,14 +57,14 @@ public static PassCodeManager getPassCodeManager() {
         return mPassCodeManagerInstance;
     }
 
-    private Long mTimestamp = 0l;
+    private Long mTimestamp = 0L;
     private int mVisibleActivitiesCounter = 0;
 
     protected PassCodeManager() {
     }
 
     public void onActivityCreated(Activity activity) {
-        if (!BuildConfig.DEBUG) {
```

Comment:
Same and in the rest of the classes where you have removed it

Reply:
there is no need of this anymore
![image](https://user-images.githubusercontent.com/3314607/54337146-49ac9d80-462e-11e9-98f8-6871c8ad40d8.png)

Reply:
That's what I mean, if is not needed anymore, please delete the `import com.owncloud.android.BuildConfig;` dependency at the top of the classes.

Reply:
Ok, this I missunderstood